packages/manager/modules/pci/src/projects/project/storages/object-storage/users/add/user-add/user-add.html

```diff
@@ -30,13 +30,18 @@
                 data-items="$ctrl.usersWithoutCredentials"
                 data-match="username"
             >
-                <span data-ng-bind="$item.userNameDescriptionKey"></span>
-                <small>
-                    <span
-                        class="float-right"
-                        data-ng-bind="$item.asCredentials"
-                    ></span>
-                </small>
+                <div class="row">
```

Comment:
Using `class="row"` without any `col-*` classes? Maybe you could use margin helpers instead?

Reply:
sure Jeremy